core/scss/utilities/variables/core/_screen-margins.scss

```diff
@@ -0,0 +1,27 @@
+@charset "UTF-8";
+
+// $screen-margins
+//
+// A sass map of screen margins for edge containment
+//
+//```
+// $screen-margins: (
+//   'xs': 20px,
+//   'sm': 30px,
+//   'md': 50px,
+//   'lg': 80px,
+//   'xl': 100px,
+//   '2xl':  150px
+// ) !default;
+// ```
+//
+// Style guide: Variables.Core.screen-margins
+//
+$screen-margins: (
+  'xs': 20px,
+  'sm': 30px,
+  'md': 50px,
+  'lg': 80px,
+  'xl': 100px,
+  '2xl':  150px
```

Comment:
Do we need screen margins for 2xl? I kind of like it better before when it's just using 100px from the xl breakpoint until it reaches 1500px.

Reply:
Yeah, we need the screen margin for 2xl or the edge of the container will touch the edge of the screen but good catch on the 150. This should be the same as xl.